bug: Release Drafter fails with 403 — insufficient GITHUB_TOKEN permissions

## Description

The **Release Drafter** workflow fails when attempting to create a draft release on `push` to `master`. The GitHub API returns a `403 Resource not accessible by integration` error.

## Error

```
##[error]Resource not accessible by integration
HttpError: Resource not accessible by integration
  status: 403
  url: https://api.github.com/repos/vmvarela/skills/releases
  x-accepted-github-permissions: contents=write; contents=write,workflows=write
```

## Root Cause

The workflow permissions block only grants `contents: read`, but creating a release requires `contents: write`:

```yaml
permissions:
  contents: read        # ← insufficient
  pull-requests: write
```

The API response header confirms the required permission:
```
x-accepted-github-permissions: contents=write
```

## Expected Behavior

The Release Drafter action should successfully create a draft release when commits are pushed to `master`.

## Proposed Fix

Update `.github/workflows/release-drafter.yml` to grant write access to contents:

```yaml
permissions:
  contents: write
  pull-requests: write
```

## Workflow Run

- **Date:** 2026-02-27T17:38:44Z
- **Runner:** ubuntu-24.04 (runner v2.331.0)
- **Action:** `release-drafter/release-drafter@v6`

--- a/.github/workflows/release-drafter.yml
+++ b/.github/workflows/release-drafter.yml
@@ -7,7 +7,7 @@ on:
     types: [opened, reopened, synchronize]
 
 permissions:
-  contents: read
+  contents: write
   pull-requests: write
 
 jobs:

